October/05/jakob_nielsen/info.yaml

```diff
@@ -0,0 +1,14 @@
+title: Jakob Nielsen
+description: |
+    On October 5, 1957 was born Jakob Nielsen.
+    He is a Danish web usability consultant.
+    Nielsen was named the “guru of Web page usability” in 1998 by The New York Times
+    and the “king of usability” by Internet Magazine.
```

Comment:
These quotes are not the standard double quotes so the test script will not catch them. IDK if they can break the bot automation. Please replace them with single quotes.

Reply:
made changes